src/components/Wallet/IndirectDeviceExtendedPublicKeyImporter.jsx

```diff
@@ -0,0 +1,263 @@
+import React from "react";
```

Comment:
We need to make this component abstract and not concrete.  Add a required prop `Reader` to the component that concrete classes (e.g. `HermitExtendedPublicKeyImporter` or `ColdcardExtendedPublicKeyImporter`) can use to pass in a custom component to handle request/parse cycle.

```
// in IndirectExtendedPublicKeyImporter.jsx

class IndirectExtendedPublicKeyImporter extends React.Component {

  render = () => {
    const {Reader} = this.props;
    return (
      <div>
        <Reader 
          hasError={this.hasBIP32PathError()} 
          onReceive={this.onReceive} 
          interaction={this.interaction()} />
      </div>
    );
  }

}

// in coldcard.js

class ColdcardExtendedPublicKeyImporter extends React.Component {

  defaultBIP32Path = () => {
    const {network, addressType} = this.props;
    // Coldcard-specific default...
  }

  render = () => {
    return (
      <IndirectExtendedPublicKeyImporter
        defaultBIP32Path={this.defaultBIP32Path()}
        Reader={ColdcardFileReader}
      />
    );
  }

}


// in hermit.js

class HermitExtendedPublicKeyImporter extends React.Component {

  render = () => {
    return (
      <IndirectExtendedPublicKeyImporter
        Reader={HermitReader}
      />
    );
  }

}
```

Reply:
will review what I did re: this more tomorrow, but I have something working for now.